test/cli/basic_commands_test.go

```diff
@@ -0,0 +1,248 @@
+	}
+}
+
+func TestCommandDocsWidth(t *testing.T) {
+	t.Parallel()
+	node := harness.NewT(t).NewNode()
+
+	// require new commands to explicitly opt in to longer lines
+	allowList := map[string]bool{
+		"ipfs add":                      true,
+		"ipfs block put":                true,
+		"ipfs daemon":                   true,
+		"ipfs config profile":           true,
+		"ipfs pin remote service":       true,
+		"ipfs name pubsub":              true,
+		"ipfs object patch":             true,
+		"ipfs swarm connect":            true,
+		"ipfs p2p forward":              true,
+		"ipfs p2p close":                true,
+		"ipfs swarm disconnect":         true,
+		"ipfs swarm addrs listen":       true,
+		"ipfs dag resolve":              true,
+		"ipfs dag get":                  true,
+		"ipfs object stat":              true,
+		"ipfs pin remote add":           true,
+		"ipfs config show":              true,
+		"ipfs config edit":              true,
+		"ipfs pin remote rm":            true,
+		"ipfs pin remote ls":            true,
+		"ipfs pin verify":               true,
+		"ipfs dht get":                  true,
+		"ipfs pin remote service add":   true,
+		"ipfs file ls":                  true,
+		"ipfs pin update":               true,
+		"ipfs pin rm":                   true,
+		"ipfs p2p":                      true,
+		"ipfs resolve":                  true,
+		"ipfs dag stat":                 true,
+		"ipfs name publish":             true,
+		"ipfs object diff":              true,
+		"ipfs object patch add-link":    true,
+		"ipfs name":                     true,
+		"ipfs object patch append-data": true,
+		"ipfs object patch set-data":    true,
+		"ipfs dht put":                  true,
+		"ipfs diag profile":             true,
+		"ipfs diag cmds":                true,
+		"ipfs swarm addrs local":        true,
+		"ipfs files ls":                 true,
+		"ipfs stats bw":                 true,
+		"ipfs urlstore add":             true,
+		"ipfs swarm peers":              true,
+		"ipfs pubsub sub":               true,
+		"ipfs repo fsck":                true,
+		"ipfs files write":              true,
+		"ipfs swarm limit":              true,
+		"ipfs commands completion fish": true,
+		"ipfs key export":               true,
+		"ipfs routing get":              true,
+		"ipfs refs":                     true,
+		"ipfs refs local":               true,
+		"ipfs cid base32":               true,
+		"ipfs pubsub pub":               true,
+		"ipfs repo ls":                  true,
+		"ipfs routing put":              true,
+		"ipfs key import":               true,
+		"ipfs swarm peering add":        true,
+		"ipfs swarm peering rm":         true,
+		"ipfs swarm peering ls":         true,
+		"ipfs update":                   true,
+		"ipfs swarm stats":              true,
+	}
+	wg := sync.WaitGroup{}
+	for _, cmd := range node.IPFSCommands() {
+		if _, ok := allowList[cmd]; ok {
+			continue
+		}
+		wg.Add(1)
+		go func(cmd string) {
```

Comment:
Instead of executing a goroutine here, we can call `t.Run` with `t.Parallel` inside.